infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/dto/ProcessorDTO.java

```diff
@@ -0,0 +1,145 @@
+package com.redhat.service.smartevents.infra.v2.api.models.dto;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.service.smartevents.infra.core.api.dto.KafkaConnectionDTO;
+import com.redhat.service.smartevents.infra.core.models.ManagedResourceStatus;
+import com.redhat.service.smartevents.infra.v2.api.models.processors.ProcessorDefinition;
+import com.redhat.service.smartevents.infra.v2.api.models.processors.ProcessorType;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ProcessorDTO {
+
+    @JsonProperty("type")
+    private ProcessorType type;
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("definition")
+    private ProcessorDefinition definition;
+
+    @JsonProperty("bridgeId")
+    private String bridgeId;
+
+    @JsonProperty("customerId")
+    private String customerId;
+
+    @JsonProperty("owner")
+    private String owner;
+
+    @JsonProperty("status")
+    private ManagedResourceStatus status;
```

Comment:
This will not exist on v2. Use of `ManagedResourceStatus` is replaced with the set of `Condition`'s.

The `Condition`'s are uni-directional; i.e. only communicated from the Operator to the Manger.